q2_demux/_summarize/assets/src/box.js

```diff
@@ -25,10 +30,15 @@ export default function plotBoxes(svg, data, x, y) {
     .selection()
     .on('mouseover', function mouseover() {
       const data = d3.select(this).data();
+      const position = data[0][0];
       const stats = data[0][1];
+      const inTheDangerZone = stats.count < seqProps.seqCount;
```

Comment:
I think this check needs to be `stats.count < seqProps.n` --- you can verify by running `summarize` with an `n` less than the total sequence count.